pinot-query-runtime/src/test/resources/queries/MathFuncs.json

```diff
@@ -435,5 +435,95 @@
         "sql": "SELECT ceil(2.0) FROM {numTbl}"
       }
     ]
+  },
+  "decimal_type_conversion": {
+    "tables": {
+      "numTbl": {
+        "schema": [
+          {"name": "intCol", "type": "INT"},
+          {"name": "longCol", "type": "LONG"},
+          {"name": "doubleCol", "type": "DOUBLE"},
+          {"name": "floatCol", "type": "FLOAT"}
+        ],
+        "inputs": [
+          [0, 0, 0.1, 3.2],
+          [123, 321, 4.2, 3.0],
+          [-456, -2, 1.1, 7.7],
+          [123, -456, 3.6, 9.1]
```

Comment:
Curious why we don't have expected results to compare against ?

Reply:
we dont need b/c we are comparing against hard-coded result, b/c we compare against a reference H2 database